week1/homework/src/package.json

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
the homework project already has a `package.json`, no need to create another one.